libsplinter/src/admin/service/messages/builders.rs

```diff
@@ -129,28 +129,28 @@ impl CreateCircuitBuilder {
     }
 
     pub fn build(self) -> Result<CreateCircuit, BuilderError> {
-        let circuit_id = self.circuit_id.ok_or_else(|| {
```

Comment:
I think there's a typo in the commit message. "When a service ID is not given to the `CreateCircuitBuilder`" should be "When a circuit ID is not given to the `CreateCircuitBuilder`"

Reply:
Fixed